src/components/GotoAnywhereModal.js

```diff
@@ -0,0 +1,372 @@
+// @flow
+
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { filter } from "fuzzaldrin-plus";
+
+import actions from "../actions";
+import {
+  getSources,
+  getGotoAnywhereEnabled,
+  getGotoAnywhereQuery,
+  getSelectedSource,
+  getSymbols
+} from "../selectors";
+import { scrollList } from "../utils/result-list";
+import { parseGotoQuery } from "../utils/gotoQuery";
+import {
+  formatSymbols,
+  formatSources
+} from "../utils/goto-anywhere-formatters";
+import Modal from "./shared/Modal";
+import SearchInput from "./shared/SearchInput";
+import ResultList from "./shared/ResultList";
+
+import type { SelectSourceOptions } from "../actions/sources";
+import type {
+  FormattedSource,
+  FormattedSymbolDeclaration,
+  FormattedSymbolDeclarations
+} from "../utils/goto-anywhere-formatters";
+import type { SourceRecord } from "../reducers/sources";
+
+type Props = {
+  enabled: boolean,
+  sources: Array<Object>,
```

Comment:
Do we have fixtures anywhere for `sources`, `selectedSource`, and `symbols` @jasonLaster?